pkg/handler/storage/saga.go

```diff
@@ -55,22 +54,17 @@ func newCreateSaga(client *Client, request *openapi.StorageV2Create) *createSaga
 	}
 }
 
-func (c *Client) generateAllocation(size int) identityapi.ResourceAllocationList {
+func (c *Client) generateAllocation(size int64) identityapi.ResourceAllocationList {
 	return identityapi.ResourceAllocationList{
 		{
 			Kind:      "filestorage",
-			Committed: size,
+			Committed: int(size),
 		},
 	}
 }
 
 func (s *createSaga) createAllocation(ctx context.Context) error {
-	convertedSize, err := strconv.Atoi(s.request.Spec.Size)
-	if err != nil {
-		return err
-	}
-
-	required := s.client.generateAllocation(convertedSize)
+	required := s.client.generateAllocation(s.request.Spec.SizeGiB)
```

Comment:
Is the allocation denominated in bytes or in GiB? Here's it's taken from the request and is denominated in GiB; in `updateAllocation` it's taken from the `regionv1.FileStorage` and it's denominated in bytes.

Reply:
Update this so that everything in k8s is stored in bytes. I could've made another conversion/helper function or leveraged the `gibtoQuantity` helper, I chose the latter. 🤷🏻‍♀️